mkosi

```diff
@@ -1240,7 +1255,7 @@ def install_boot_loader_arch(args, workspace):
     kernel_version = next(filter(lambda x: x[0].isdigit(), os.listdir(os.path.join(workspace, "root", "lib/modules"))))
 
     run_workspace_command(args, workspace,
-                      "/usr/bin/kernel-install", "add", kernel_version, "/boot/vmlinuz-linux")
+                      "/usr/bin/kernel-install", "add", kernel_version, glob.glob("/boot/vmlinuz-*")[0])
```

Comment:
hmm, this is ugly, given that it silently picks whatever is first in the globbing list... could you hide this in a new function find_vmlinuz() or so, which hides this, and prints a warning if the kernel image name is not unique, and indicates which one of the options it is picking? (even better: make this deterministic, by sorting the list of kernel images)

Reply:
Did so, as for the warning, i'm using print() as there is no proper logging setup.